server/plugin_test.go

```diff
@@ -0,0 +1,121 @@
+package main
+
+import (
+	"encoding/json"
+	"strings"
+	"testing"
+	"time"
+
+	"github.com/cristalhq/jwt/v2"
+	"github.com/mattermost/mattermost-server/v5/model"
+	"github.com/mattermost/mattermost-server/v5/plugin/plugintest"
+	"github.com/stretchr/testify/mock"
+	"github.com/stretchr/testify/require"
+)
+
+func TestVerifyJwt(t *testing.T) {
+	testToken := "eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJzdWIiOiIxMjM0NTY3ODkwIiwibmFtZSI6IkpvaG4gRG9lIiwiaWF0IjoxNTE2MjM5MDIyfQ.mpHl842O7xEZjgQ8CyX8xYLDoEORGVMnAxULkW-u8Ek"
+	t.Run("bad secret should fail", func(t *testing.T) {
+		claims, err := verifyJwt("bad-secret", testToken)
+		require.NotNil(t, err)
+		require.Nil(t, claims)
+	})
+	t.Run("right secret should work", func(t *testing.T) {
+		claims, err := verifyJwt("test-secret", testToken)
+		require.Nil(t, err)
+		require.Equal(t, "1234567890", claims.Subject)
+		issuedAt, err := claims.IssuedAt.MarshalJSON()
+		require.Nil(t, err)
+		require.Equal(t, "1516239022", string(issuedAt))
```

Comment:
Probably not needed since we are using them just once, but wonder if we should have all these variables (`test-secret`, `1234567890`, ...) named and predefined (at the same level as testToken).